interfaces/src/db.ts

```diff
@@ -87,3 +93,55 @@ export interface Query {
   skip?: number;
   orderBy?: ReadonlyArray<Order>;
 }
+
+export interface ClientContext {
+  debugId: string;
+}
+
+export interface ServerOnlyContext {
+  tags?: { [key: string]: string };
+  logLevel?: string;
+  logExtra?: { [key: string]: any };
+  sampleRate?: number;
+}
+
+export type Version = [number, number];
+
+export interface VersionedObject {
+  version: Version;
+  value: Serializable;
+}
+
+interface Patches {
+  /**
+   * Stores changes made to the document, meant to be used internally by poll().
+   */
+  patches: ReadonlyArray<Patch>;
+  updatedAt: number;
+}
+
+export interface StoredDocument extends VersionedObject, Patches {}
+
+export interface Tombstone extends Patches {
+  version: Version;
+}
+
+export interface DB {
+  /**
+   * Gets a single document.
+   * @return - value or undefined if key doesn’t exist.
+   */
+  get: {
+    params: { key: string; };
+    returns: Serializable | undefined;
+  };
+
+  /**
+   * Gets a single document with its version.
+   * @return - { version, value } or undefined if key doesn’t exist.
+   */
+  getWithMeta: {
```

Comment:
Not part of the public interface, should be `startPolling` (see `db/src/index.ts`).

Reply:
This is a bit odd, that file wraps the DB interface to give the client.  E.g. even `update` etc. in that file don't take a `db` argument.  Not doing the rename until we better understand where to draw the line.

Reply:
We agreed we should use `startPolling` when we talked about this as this is the method that is supposed to signal the backend to prepare the subscription stream.